app/import_functions.go

```diff
@@ -1281,14 +1292,26 @@ func (a *App) importMultiplePostLines(lines []LineImportWorkerData, dryRun bool)
 	}
 
 	if len(postsForCreateList) > 0 {
-		if _, idx, err := a.Srv().Store.Post().SaveMultiple(postsForCreateList); err != nil {
+		if _, idx, nErr := a.Srv().Store.Post().SaveMultiple(postsForCreateList); nErr != nil {
+			var appErr *model.AppError
+			var invErr *store.ErrInvalidInput
+			var retErr *model.AppError
+			switch {
+			case errors.As(err, &appErr):
```

Comment:
`errors.As(nErr, <>)`. This is there in multiple places.

Reply:
Sorry I didn't understand this suggestions. Did you mean we have code duplication ?

Reply:
This seems fixed with changes I made so I'm goint to mark it as revolved :)